[2020-01-22 15:07:54 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1376)
[2020-01-22 15:07:54 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1377)
[2020-01-22 15:07:54 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1378)
[2020-01-22 15:07:54 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1379)
[2020-01-22 15:07:55 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1380)
[2020-01-22 15:07:55 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1381)
[2020-01-22 15:07:55 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1382)
[2020-01-22 15:07:55 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1383)
[2020-01-22 15:07:55 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1384)
[2020-01-22 15:07:55 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1385)
[2020-01-22 15:07:55 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1386)
[2020-01-22 15:07:55 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1387)
[2020-01-22 15:07:55 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1388)
[2020-01-22 15:07:55 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1389)
[2020-01-22 15:07:55 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1390)
[2020-01-22 15:07:55 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1391)
[2020-01-22 15:07:55 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1392)
[2020-01-22 15:24:41 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1483)
[2020-01-22 15:24:41 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1484)
[2020-01-22 15:24:41 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1485)
[2020-01-22 15:24:41 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1486)
[2020-01-22 15:24:42 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1487)
[2020-01-22 15:24:42 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1488)
[2020-01-22 15:24:42 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1489)
[2020-01-22 15:24:42 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1490)
[2020-01-22 15:24:42 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1491)
[2020-01-22 15:24:42 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1492)
[2020-01-22 15:24:42 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1493)
[2020-01-22 15:24:42 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1494)
[2020-01-22 15:24:42 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1495)
[2020-01-22 15:24:42 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1496)
[2020-01-22 15:24:42 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1497)
[2020-01-22 15:24:42 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1498)
[2020-01-22 15:24:42 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1499)
[2020-01-22 15:31:13 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1503)
[2020-01-22 15:31:13 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1504)
[2020-01-22 15:31:13 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1505)
[2020-01-22 15:31:13 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1506)
[2020-01-22 15:31:14 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1507)
[2020-01-22 15:31:14 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1508)
[2020-01-22 15:31:14 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1509)
[2020-01-22 15:31:14 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1510)
[2020-01-22 15:31:14 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1511)
[2020-01-22 15:31:14 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1512)
[2020-01-22 15:31:14 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1513)
[2020-01-22 15:31:14 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1514)
[2020-01-22 15:31:14 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1515)
[2020-01-22 15:31:14 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1516)
[2020-01-22 15:31:14 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1517)
[2020-01-22 15:31:14 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1518)
[2020-01-22 15:31:14 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1519)
[2020-01-22 15:32:19 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1540)
[2020-01-22 15:32:19 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1541)
[2020-01-22 15:32:19 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1542)
[2020-01-22 15:32:19 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1543)
[2020-01-22 15:32:19 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1544)
[2020-01-22 15:32:19 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1545)
[2020-01-22 15:32:20 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1546)
[2020-01-22 15:32:20 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1547)
[2020-01-22 15:32:20 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1548)
[2020-01-22 15:32:20 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1549)
[2020-01-22 15:32:20 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1550)
[2020-01-22 15:32:20 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1551)
[2020-01-22 15:32:20 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1552)
[2020-01-22 15:32:20 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1553)
[2020-01-22 15:32:20 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1554)
[2020-01-22 15:32:20 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1555)
[2020-01-22 15:32:20 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1556)
[2020-01-22 16:10:40 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1583)
[2020-01-22 16:10:40 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1584)
[2020-01-22 16:10:40 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1585)
[2020-01-22 16:10:40 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1586)
[2020-01-22 16:10:40 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1587)
[2020-01-22 16:10:40 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1588)
[2020-01-22 16:10:41 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1589)
[2020-01-22 16:10:41 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1590)
[2020-01-22 16:10:41 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1591)
[2020-01-22 16:10:41 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1592)
[2020-01-22 16:10:41 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1593)
[2020-01-22 16:10:41 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1594)
[2020-01-22 16:10:41 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1595)
[2020-01-22 16:10:41 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1596)
[2020-01-22 16:10:41 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1597)
[2020-01-22 16:10:41 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1598)
[2020-01-22 16:10:41 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1599)
[2020-01-22 16:27:59 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1656)
[2020-01-22 16:27:59 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1657)
[2020-01-22 16:27:59 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1658)
[2020-01-22 16:27:59 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1660)
[2020-01-22 16:27:59 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1661)
[2020-01-22 16:28:00 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1662)
[2020-01-22 16:28:00 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1663)
[2020-01-22 16:28:00 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1664)
[2020-01-22 16:28:00 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1665)
[2020-01-22 16:28:00 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1666)
[2020-01-22 16:28:00 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1667)
[2020-01-22 16:28:00 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1668)
[2020-01-22 16:28:00 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1669)
[2020-01-22 16:28:00 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1670)
[2020-01-22 16:28:00 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1671)
[2020-01-22 16:28:00 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1672)
[2020-01-22 16:28:00 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1673)
[2020-01-22 16:47:22 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1687)
[2020-01-22 16:47:22 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1688)
[2020-01-22 16:47:22 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1689)
[2020-01-22 16:47:22 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1690)
[2020-01-22 16:47:22 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1691)
[2020-01-22 16:47:22 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1692)
[2020-01-22 16:47:23 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1693)
[2020-01-22 16:47:23 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1694)
[2020-01-22 16:47:23 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1695)
[2020-01-22 16:47:23 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1696)
[2020-01-22 16:47:23 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1697)
[2020-01-22 16:47:23 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1698)
[2020-01-22 16:47:23 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1699)
[2020-01-22 16:47:23 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1700)
[2020-01-22 16:47:23 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1701)
[2020-01-22 16:47:23 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1702)
[2020-01-22 16:47:23 -0800] [1373] [CRITICAL] WORKER TIMEOUT (pid:1703)
[2020-02-14 03:28:00 -0800] [47181] [CRITICAL] WORKER TIMEOUT (pid:47188)
[2020-02-14 09:48:14 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2167)
[2020-02-14 09:48:15 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2171)
[2020-02-14 09:48:15 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2172)
[2020-02-14 09:48:15 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2174)
[2020-02-14 09:48:15 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2176)
[2020-02-14 09:48:15 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2179)
[2020-02-14 09:48:15 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2181)
[2020-02-14 09:48:16 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2170)
[2020-02-14 09:48:16 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2175)
[2020-02-14 09:48:16 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2183)
[2020-02-14 09:49:30 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2168)
[2020-02-14 09:49:30 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2169)
[2020-02-14 09:49:30 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2173)
[2020-02-14 09:49:30 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2177)
[2020-02-14 09:49:30 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2178)
[2020-02-14 09:49:31 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2180)
[2020-02-14 09:49:31 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2182)
[2020-02-14 09:49:31 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2226)
[2020-02-14 09:49:31 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2227)
[2020-02-14 09:49:31 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2228)
[2020-02-14 09:49:31 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2229)
[2020-02-14 09:49:31 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2230)
[2020-02-14 09:49:31 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2231)
[2020-02-14 09:49:31 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2232)
[2020-02-14 09:49:33 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2235)
[2020-02-14 09:49:33 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2236)
[2020-02-14 09:49:33 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2237)
[2020-02-14 10:03:16 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2263)
[2020-02-14 10:03:16 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2264)
[2020-02-14 10:03:16 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2265)
[2020-02-14 10:03:16 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2266)
[2020-02-14 10:03:16 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2267)
[2020-02-14 10:03:16 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2268)
[2020-02-14 10:03:17 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2269)
[2020-02-14 10:03:17 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2270)
[2020-02-14 10:03:17 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2271)
[2020-02-14 10:03:17 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2272)
[2020-02-14 10:03:17 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2273)
[2020-02-14 10:03:17 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2274)
[2020-02-14 10:03:17 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2275)
[2020-02-14 10:03:17 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2276)
[2020-02-14 10:03:17 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2278)
[2020-02-14 10:03:17 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2279)
[2020-02-14 10:03:17 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2280)
[2020-02-14 10:08:18 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2300)
[2020-02-14 10:08:18 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2301)
[2020-02-14 10:08:18 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2302)
[2020-02-14 10:08:18 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2303)
[2020-02-14 10:08:18 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2304)
[2020-02-14 10:08:19 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2305)
[2020-02-14 10:08:19 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2306)
[2020-02-14 10:08:19 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2307)
[2020-02-14 10:08:19 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2308)
[2020-02-14 10:08:19 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2309)
[2020-02-14 10:08:19 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2310)
[2020-02-14 10:08:19 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2311)
[2020-02-14 10:08:19 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2312)
[2020-02-14 10:08:19 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2313)
[2020-02-14 10:08:19 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2314)
[2020-02-14 10:08:19 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2315)
[2020-02-14 10:08:19 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2316)
[2020-02-14 10:17:22 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2328)
[2020-02-14 10:17:22 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2329)
[2020-02-14 10:17:22 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2330)
[2020-02-14 10:17:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2331)
[2020-02-14 10:17:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2332)
[2020-02-14 10:17:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2333)
[2020-02-14 10:17:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2334)
[2020-02-14 10:17:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2335)
[2020-02-14 10:17:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2336)
[2020-02-14 10:17:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2337)
[2020-02-14 10:17:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2338)
[2020-02-14 10:17:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2339)
[2020-02-14 10:17:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2340)
[2020-02-14 10:17:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2341)
[2020-02-14 10:17:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2342)
[2020-02-14 10:17:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2343)
[2020-02-14 10:17:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2344)
[2020-02-14 10:39:49 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2400)
[2020-02-14 10:39:49 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2401)
[2020-02-14 10:39:49 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2402)
[2020-02-14 10:39:49 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2403)
[2020-02-14 10:39:50 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2404)
[2020-02-14 10:39:50 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2405)
[2020-02-14 10:39:50 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2406)
[2020-02-14 10:39:50 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2407)
[2020-02-14 10:39:50 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2408)
[2020-02-14 10:39:50 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2409)
[2020-02-14 10:39:50 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2410)
[2020-02-14 10:39:50 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2411)
[2020-02-14 10:39:50 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2412)
[2020-02-14 10:39:50 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2413)
[2020-02-14 10:39:50 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2414)
[2020-02-14 10:39:50 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2415)
[2020-02-14 10:39:50 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2416)
[2020-02-14 11:48:41 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2864)
[2020-02-14 11:48:41 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2865)
[2020-02-14 11:48:41 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2866)
[2020-02-14 11:48:41 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2867)
[2020-02-14 11:48:41 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2868)
[2020-02-14 11:48:42 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2869)
[2020-02-14 11:48:42 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2870)
[2020-02-14 11:48:42 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2871)
[2020-02-14 11:48:42 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2872)
[2020-02-14 11:48:42 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2873)
[2020-02-14 11:48:42 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2874)
[2020-02-14 11:48:42 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2875)
[2020-02-14 11:48:42 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2876)
[2020-02-14 11:48:42 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2877)
[2020-02-14 11:48:42 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2878)
[2020-02-14 11:48:42 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2879)
[2020-02-14 11:48:42 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2880)
[2020-02-14 12:26:22 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2900)
[2020-02-14 12:26:22 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2901)
[2020-02-14 12:26:22 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2902)
[2020-02-14 12:26:22 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2903)
[2020-02-14 12:26:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2904)
[2020-02-14 12:26:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2905)
[2020-02-14 12:26:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2906)
[2020-02-14 12:26:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2907)
[2020-02-14 12:26:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2908)
[2020-02-14 12:26:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2909)
[2020-02-14 12:26:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2910)
[2020-02-14 12:26:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2911)
[2020-02-14 12:26:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2912)
[2020-02-14 12:26:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2913)
[2020-02-14 12:26:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2914)
[2020-02-14 12:26:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2915)
[2020-02-14 12:26:23 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:2916)
[2020-02-14 12:38:11 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3005)
[2020-02-14 12:38:11 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3006)
[2020-02-14 12:38:11 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3007)
[2020-02-14 12:38:11 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3008)
[2020-02-14 12:38:12 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3009)
[2020-02-14 12:38:12 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3010)
[2020-02-14 12:38:12 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3011)
[2020-02-14 12:38:12 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3012)
[2020-02-14 12:38:12 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3013)
[2020-02-14 12:38:12 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3014)
[2020-02-14 12:38:12 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3015)
[2020-02-14 12:38:12 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3016)
[2020-02-14 12:38:12 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3017)
[2020-02-14 12:38:12 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3018)
[2020-02-14 12:38:12 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3019)
[2020-02-14 12:38:12 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3020)
[2020-02-14 12:38:12 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3021)
[2020-02-14 13:39:59 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3043)
[2020-02-14 13:39:59 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3044)
[2020-02-14 13:39:59 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3045)
[2020-02-14 13:39:59 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3046)
[2020-02-14 13:40:00 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3047)
[2020-02-14 14:05:59 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3048)
[2020-02-14 14:05:59 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3049)
[2020-02-14 14:05:59 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3050)
[2020-02-14 14:05:59 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3051)
[2020-02-14 14:06:00 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3052)
[2020-02-14 14:06:00 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3053)
[2020-02-14 14:06:00 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3054)
[2020-02-14 14:06:00 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3055)
[2020-02-14 14:06:00 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3056)
[2020-02-14 14:06:00 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3057)
[2020-02-14 14:06:00 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3058)
[2020-02-14 14:06:00 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3059)
[2020-02-14 14:06:00 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3072)
[2020-02-14 14:06:00 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3073)
[2020-02-14 14:06:00 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3074)
[2020-02-14 14:06:00 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3075)
[2020-02-14 14:06:00 -0800] [2164] [CRITICAL] WORKER TIMEOUT (pid:3076)
[2020-02-14 14:52:22 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3240)
[2020-02-14 14:52:22 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3241)
[2020-02-14 14:52:22 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3242)
[2020-02-14 14:52:22 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3243)
[2020-02-14 14:52:22 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3244)
[2020-02-14 14:52:22 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3245)
[2020-02-14 14:52:23 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3246)
[2020-02-14 14:52:23 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3247)
[2020-02-14 14:52:23 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3248)
[2020-02-14 14:52:23 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3249)
[2020-02-14 14:52:23 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3250)
[2020-02-14 14:52:23 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3251)
[2020-02-14 14:52:23 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3252)
[2020-02-14 14:52:23 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3253)
[2020-02-14 14:52:23 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3254)
[2020-02-14 14:52:23 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3255)
[2020-02-14 14:52:23 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3256)
[2020-02-14 15:29:20 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3341)
[2020-02-14 15:29:20 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3343)
[2020-02-14 15:31:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3342)
[2020-02-14 15:31:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3345)
[2020-02-14 15:31:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3346)
[2020-02-14 15:31:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3347)
[2020-02-14 15:31:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3348)
[2020-02-14 15:31:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3349)
[2020-02-14 15:31:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3350)
[2020-02-14 15:31:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3351)
[2020-02-14 15:31:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3352)
[2020-02-14 15:31:05 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3353)
[2020-02-14 15:31:05 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3354)
[2020-02-14 15:31:05 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3355)
[2020-02-14 15:31:05 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3356)
[2020-02-14 15:31:05 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3357)
[2020-02-14 15:31:05 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3358)
[2020-02-14 15:31:05 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3426)
[2020-02-14 15:31:05 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3427)
[2020-02-14 15:32:49 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3436)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3437)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3438)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3439)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3440)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3441)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3442)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3443)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3444)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3445)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3446)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3447)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3448)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3449)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3450)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3451)
[2020-02-14 15:32:50 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3452)
[2020-02-14 15:41:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3486)
[2020-02-14 15:41:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3487)
[2020-02-14 15:41:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3488)
[2020-02-14 15:41:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3489)
[2020-02-14 15:41:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3491)
[2020-02-14 15:41:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3492)
[2020-02-14 15:41:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3493)
[2020-02-14 15:41:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3494)
[2020-02-14 15:41:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3495)
[2020-02-14 15:41:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3496)
[2020-02-14 15:41:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3497)
[2020-02-14 15:41:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3498)
[2020-02-14 15:41:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3499)
[2020-02-14 15:41:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3500)
[2020-02-14 15:41:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3501)
[2020-02-14 15:41:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3502)
[2020-02-14 15:41:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3503)
[2020-02-14 15:52:47 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3524)
[2020-02-14 15:52:47 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3525)
[2020-02-14 15:52:47 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3526)
[2020-02-14 15:52:47 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3527)
[2020-02-14 15:52:48 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3528)
[2020-02-14 15:52:48 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3529)
[2020-02-14 15:52:48 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3530)
[2020-02-14 15:52:48 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3531)
[2020-02-14 15:52:48 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3532)
[2020-02-14 15:52:48 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3533)
[2020-02-14 15:52:48 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3534)
[2020-02-14 15:52:48 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3535)
[2020-02-14 15:52:48 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3536)
[2020-02-14 15:52:48 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3537)
[2020-02-14 15:52:48 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3538)
[2020-02-14 15:52:48 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3539)
[2020-02-14 15:52:48 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3540)
[2020-02-14 15:56:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3579)
[2020-02-14 15:56:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3580)
[2020-02-14 15:56:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3581)
[2020-02-14 15:56:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3582)
[2020-02-14 15:56:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3583)
[2020-02-14 15:56:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3585)
[2020-02-14 15:56:32 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3586)
[2020-02-14 15:56:32 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3587)
[2020-02-14 15:56:32 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3588)
[2020-02-14 15:56:32 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3589)
[2020-02-14 15:56:32 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3590)
[2020-02-14 15:56:32 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3591)
[2020-02-14 15:56:32 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3592)
[2020-02-14 15:56:32 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3593)
[2020-02-14 15:56:32 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3594)
[2020-02-14 15:56:32 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3595)
[2020-02-14 15:56:32 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3596)
[2020-02-14 17:03:29 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3620)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3621)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3622)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3623)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3624)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3625)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3626)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3627)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3628)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3629)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3630)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3631)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3632)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3633)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3634)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3635)
[2020-02-14 17:03:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3636)
[2020-02-14 17:09:16 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3753)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3754)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3755)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3756)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3757)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3758)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3759)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3760)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3761)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3762)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3763)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3764)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3765)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3766)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3767)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3768)
[2020-02-14 17:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3769)
[2020-02-14 17:23:02 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3793)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3794)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3795)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3796)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3797)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3798)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3799)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3800)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3801)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3802)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3803)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3804)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3805)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3806)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3807)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3808)
[2020-02-14 17:23:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3809)
[2020-02-14 17:24:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3829)
[2020-02-14 17:24:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3830)
[2020-02-14 17:24:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3831)
[2020-02-14 17:24:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3832)
[2020-02-14 17:24:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3833)
[2020-02-14 17:24:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3834)
[2020-02-14 17:24:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3835)
[2020-02-14 17:24:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3836)
[2020-02-14 17:24:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3837)
[2020-02-14 17:24:30 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3838)
[2020-02-14 17:24:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3839)
[2020-02-14 17:24:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3840)
[2020-02-14 17:24:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3841)
[2020-02-14 17:24:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3842)
[2020-02-14 17:24:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3843)
[2020-02-14 17:24:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3844)
[2020-02-14 17:24:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3845)
[2020-02-14 19:25:07 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3856)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3857)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3858)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3859)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3860)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3861)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3862)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3863)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3864)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3865)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3866)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3867)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3868)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3869)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3870)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3871)
[2020-02-14 19:25:08 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3872)
[2020-02-14 21:25:23 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3881)
[2020-02-14 21:25:24 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3891)
[2020-02-14 21:25:24 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3892)
[2020-02-14 21:25:24 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3893)
[2020-02-14 21:25:24 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3894)
[2020-02-14 21:25:24 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3895)
[2020-02-14 21:25:24 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3896)
[2020-02-14 21:25:24 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3897)
[2020-02-14 21:41:45 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3882)
[2020-02-14 21:41:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3883)
[2020-02-14 21:41:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3884)
[2020-02-14 21:41:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3885)
[2020-02-14 21:41:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3886)
[2020-02-14 21:41:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3887)
[2020-02-14 21:41:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3888)
[2020-02-14 21:41:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3889)
[2020-02-14 21:41:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3890)
[2020-02-15 00:05:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3900)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3901)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3902)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3903)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3904)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3905)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3906)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3907)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3912)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3913)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3914)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3915)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3916)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3917)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3918)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3919)
[2020-02-15 00:05:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:3920)
[2020-02-15 02:09:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4016)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4017)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4018)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4019)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4020)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4021)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4022)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4023)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4024)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4025)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4026)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4027)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4028)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4029)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4030)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4031)
[2020-02-15 02:09:18 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4032)
[2020-02-15 02:45:05 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4045)
[2020-02-15 02:45:06 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4046)
[2020-02-15 02:45:06 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4047)
[2020-02-15 02:45:06 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4048)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4049)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4051)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4052)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4053)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4054)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4055)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4056)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4057)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4058)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4059)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4060)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4061)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4062)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4071)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4072)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4073)
[2020-02-15 05:11:13 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4074)
[2020-02-15 05:26:34 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4088)
[2020-02-15 05:26:34 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4089)
[2020-02-15 05:26:34 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4090)
[2020-02-15 05:26:34 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4091)
[2020-02-15 05:26:34 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4092)
[2020-02-15 05:26:34 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4093)
[2020-02-15 05:26:34 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4094)
[2020-02-15 06:11:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4095)
[2020-02-15 06:11:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4096)
[2020-02-15 06:11:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4097)
[2020-02-15 06:11:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4098)
[2020-02-15 06:11:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4099)
[2020-02-15 06:11:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4100)
[2020-02-15 06:11:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4101)
[2020-02-15 06:11:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4103)
[2020-02-15 06:11:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4105)
[2020-02-15 06:11:14 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4106)
[2020-02-15 06:11:15 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4151)
[2020-02-15 06:11:15 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4152)
[2020-02-15 06:11:15 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4153)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4147)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4148)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4149)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4150)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4157)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4158)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4159)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4160)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4161)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4162)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4163)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4164)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4165)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4166)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4167)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4168)
[2020-02-15 08:41:21 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4169)
[2020-02-15 09:17:52 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4207)
[2020-02-15 09:45:36 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4208)
[2020-02-15 09:45:36 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4209)
[2020-02-15 09:45:36 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4210)
[2020-02-15 09:45:36 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4211)
[2020-02-15 09:45:36 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4212)
[2020-02-15 09:45:36 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4213)
[2020-02-15 09:45:37 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4282)
[2020-02-15 10:00:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4214)
[2020-02-15 10:00:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4215)
[2020-02-15 10:00:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4216)
[2020-02-15 10:00:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4217)
[2020-02-15 10:00:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4218)
[2020-02-15 10:00:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4219)
[2020-02-15 10:00:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4220)
[2020-02-15 10:00:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4221)
[2020-02-15 10:00:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4222)
[2020-02-15 10:00:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4223)
[2020-02-15 10:07:02 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4361)
[2020-02-15 10:07:02 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4371)
[2020-02-15 10:07:02 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4372)
[2020-02-15 10:07:02 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4373)
[2020-02-15 10:07:02 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4374)
[2020-02-15 10:07:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4375)
[2020-02-15 10:07:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4376)
[2020-02-15 10:07:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4377)
[2020-02-15 10:07:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4378)
[2020-02-15 10:07:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4379)
[2020-02-15 10:07:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4380)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4362)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4363)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4364)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4365)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4366)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4367)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4401)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4403)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4404)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4405)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4406)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4407)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4408)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4409)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4410)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4411)
[2020-02-15 10:09:31 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4412)
[2020-02-15 12:03:16 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4427)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4429)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4430)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4431)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4432)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4433)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4434)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4435)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4436)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4437)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4438)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4439)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4440)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4441)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4442)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4443)
[2020-02-15 12:03:17 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4444)
[2020-02-15 12:05:27 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4636)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4637)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4638)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4639)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4640)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4641)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4642)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4643)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4644)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4645)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4646)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4647)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4648)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4649)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4650)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4651)
[2020-02-15 12:05:28 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4652)
[2020-02-15 12:06:45 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4676)
[2020-02-15 12:06:45 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4677)
[2020-02-15 12:06:45 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4678)
[2020-02-15 12:06:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4679)
[2020-02-15 12:06:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4680)
[2020-02-15 12:06:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4681)
[2020-02-15 12:06:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4682)
[2020-02-15 12:06:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4683)
[2020-02-15 12:06:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4684)
[2020-02-15 12:06:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4685)
[2020-02-15 12:06:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4686)
[2020-02-15 12:06:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4687)
[2020-02-15 12:06:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4688)
[2020-02-15 12:06:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4689)
[2020-02-15 12:06:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4690)
[2020-02-15 12:06:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4691)
[2020-02-15 12:06:46 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4692)
[2020-02-15 12:19:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4722)
[2020-02-15 12:19:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4723)
[2020-02-15 12:19:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4724)
[2020-02-15 12:19:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4725)
[2020-02-15 12:19:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4726)
[2020-02-15 12:19:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4727)
[2020-02-15 12:19:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4728)
[2020-02-15 12:19:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4729)
[2020-02-15 12:19:03 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4730)
[2020-02-15 12:19:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4731)
[2020-02-15 12:19:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4732)
[2020-02-15 12:19:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4733)
[2020-02-15 12:19:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4734)
[2020-02-15 12:19:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4735)
[2020-02-15 12:19:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4736)
[2020-02-15 12:19:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4737)
[2020-02-15 12:19:04 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4738)
[2020-02-15 13:15:58 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4752)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4753)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4754)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4755)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4756)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4757)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4758)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4759)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4760)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4761)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4762)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4763)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4764)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4765)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4766)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4767)
[2020-02-15 13:15:59 -0800] [3237] [CRITICAL] WORKER TIMEOUT (pid:4768)
[2020-04-19 15:18:55 -0700] [39352] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:18:55 -0700] [39352] [ERROR] Retrying in 1 second.
[2020-04-19 15:18:56 -0700] [39352] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:18:56 -0700] [39352] [ERROR] Retrying in 1 second.
[2020-04-19 15:18:57 -0700] [39352] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:18:57 -0700] [39352] [ERROR] Retrying in 1 second.
[2020-04-19 15:18:58 -0700] [39352] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:18:58 -0700] [39352] [ERROR] Retrying in 1 second.
[2020-04-19 15:18:59 -0700] [39352] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:18:59 -0700] [39352] [ERROR] Retrying in 1 second.
[2020-04-19 15:19:00 -0700] [39352] [ERROR] Can't connect to ('76.102.164.115', 22362)
[2020-04-19 15:19:03 -0700] [39366] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:19:03 -0700] [39366] [ERROR] Retrying in 1 second.
[2020-04-19 15:19:04 -0700] [39366] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:19:04 -0700] [39366] [ERROR] Retrying in 1 second.
[2020-04-19 15:19:05 -0700] [39366] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:19:05 -0700] [39366] [ERROR] Retrying in 1 second.
[2020-04-19 15:19:06 -0700] [39366] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:19:06 -0700] [39366] [ERROR] Retrying in 1 second.
[2020-04-19 15:19:07 -0700] [39366] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:19:07 -0700] [39366] [ERROR] Retrying in 1 second.
[2020-04-19 15:19:08 -0700] [39366] [ERROR] Can't connect to ('76.102.164.115', 22362)
[2020-04-19 15:20:46 -0700] [39544] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:20:46 -0700] [39544] [ERROR] Retrying in 1 second.
[2020-04-19 15:20:47 -0700] [39544] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:20:47 -0700] [39544] [ERROR] Retrying in 1 second.
[2020-04-19 15:20:48 -0700] [39544] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:20:48 -0700] [39544] [ERROR] Retrying in 1 second.
[2020-04-19 15:20:49 -0700] [39544] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:20:49 -0700] [39544] [ERROR] Retrying in 1 second.
[2020-04-19 15:20:50 -0700] [39544] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:20:50 -0700] [39544] [ERROR] Retrying in 1 second.
[2020-04-19 15:20:51 -0700] [39544] [ERROR] Can't connect to ('76.102.164.115', 22362)
[2020-04-19 15:22:38 -0700] [39622] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:22:38 -0700] [39622] [ERROR] Retrying in 1 second.
[2020-04-19 15:22:39 -0700] [39622] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:22:39 -0700] [39622] [ERROR] Retrying in 1 second.
[2020-04-19 15:22:40 -0700] [39622] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:22:40 -0700] [39622] [ERROR] Retrying in 1 second.
[2020-04-19 15:22:41 -0700] [39622] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:22:41 -0700] [39622] [ERROR] Retrying in 1 second.
[2020-04-19 15:22:42 -0700] [39622] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 15:22:42 -0700] [39622] [ERROR] Retrying in 1 second.
[2020-04-19 15:22:43 -0700] [39622] [ERROR] Can't connect to ('76.102.164.115', 22362)
[2020-04-19 16:11:19 -0700] [40181] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 16:11:19 -0700] [40181] [ERROR] Retrying in 1 second.
[2020-04-19 16:11:20 -0700] [40181] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 16:11:20 -0700] [40181] [ERROR] Retrying in 1 second.
[2020-04-19 16:11:21 -0700] [40181] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 16:11:21 -0700] [40181] [ERROR] Retrying in 1 second.
[2020-04-19 16:11:22 -0700] [40181] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 16:11:22 -0700] [40181] [ERROR] Retrying in 1 second.
[2020-04-19 16:11:23 -0700] [40181] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 16:11:23 -0700] [40181] [ERROR] Retrying in 1 second.
[2020-04-19 16:11:23 -0700] [40195] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 16:11:23 -0700] [40195] [ERROR] Retrying in 1 second.
[2020-04-19 16:11:24 -0700] [40181] [ERROR] Can't connect to ('76.102.164.115', 22362)
[2020-04-19 16:11:24 -0700] [40195] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 16:11:24 -0700] [40195] [ERROR] Retrying in 1 second.
[2020-04-19 16:11:25 -0700] [40195] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 16:11:25 -0700] [40195] [ERROR] Retrying in 1 second.
[2020-04-19 16:11:26 -0700] [40195] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 16:11:26 -0700] [40195] [ERROR] Retrying in 1 second.
[2020-04-19 16:11:27 -0700] [40195] [ERROR] Invalid address: ('76.102.164.115', 22362)
[2020-04-19 16:11:27 -0700] [40195] [ERROR] Retrying in 1 second.
[2020-04-19 16:11:28 -0700] [40195] [ERROR] Can't connect to ('76.102.164.115', 22362)
[2020-04-19 18:59:45 -0700] [40870] [CRITICAL] WORKER TIMEOUT (pid:40905)
[2020-04-23 11:44:43 -0700] [5698] [CRITICAL] WORKER TIMEOUT (pid:5706)
[2020-04-24 00:55:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10304)
[2020-04-24 01:45:02 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10302)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10303)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10305)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10306)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10307)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10308)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10309)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10310)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10311)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10312)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10313)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10314)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10315)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10316)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10317)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10318)
[2020-04-24 01:45:03 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10589)
[2020-04-24 08:14:58 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10602)
[2020-04-24 11:03:04 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10614)
[2020-04-24 17:58:58 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10603)
[2020-04-24 17:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10604)
[2020-04-24 17:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10605)
[2020-04-24 17:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10606)
[2020-04-24 17:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10607)
[2020-04-24 17:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10608)
[2020-04-24 17:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10609)
[2020-04-24 17:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10610)
[2020-04-24 17:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10611)
[2020-04-24 17:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10612)
[2020-04-24 17:58:58 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10613)
[2020-04-24 17:58:58 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10615)
[2020-04-24 17:58:58 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10616)
[2020-04-24 17:58:58 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10617)
[2020-04-24 17:58:58 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10618)
[2020-04-24 17:58:58 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10803)
[2020-04-24 17:58:58 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:10816)
[2020-04-24 18:58:58 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11725)
[2020-04-24 18:58:58 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11726)
[2020-04-24 18:58:58 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11727)
[2020-04-24 18:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11728)
[2020-04-24 18:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11729)
[2020-04-24 18:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11730)
[2020-04-24 18:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11731)
[2020-04-24 18:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11732)
[2020-04-24 18:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11733)
[2020-04-24 18:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11734)
[2020-04-24 18:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11735)
[2020-04-24 18:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11736)
[2020-04-24 18:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11737)
[2020-04-24 18:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11738)
[2020-04-24 18:58:59 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11739)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11740)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11741)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11745)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11746)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11747)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11748)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11749)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11750)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11751)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11752)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11753)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11754)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11755)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11756)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11757)
[2020-04-24 22:26:33 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11758)
[2020-04-24 22:26:32 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11759)
[2020-04-24 22:57:18 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11793)
[2020-04-24 22:57:18 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11794)
[2020-04-24 22:57:18 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11795)
[2020-04-24 22:57:18 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11796)
[2020-04-24 22:57:18 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11797)
[2020-04-24 22:57:18 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11798)
[2020-04-24 22:57:18 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11799)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11800)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11801)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11802)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11803)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11804)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11805)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11806)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11807)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11808)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11809)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11815)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11816)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11817)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11818)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11820)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11821)
[2020-04-25 03:23:35 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11822)
[2020-04-25 12:27:34 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11852)
[2020-04-26 08:03:20 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11849)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11850)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11851)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11853)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11854)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11855)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11856)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11857)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11858)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11859)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11860)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11861)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11862)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11863)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11871)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:11872)
[2020-04-26 08:03:21 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:12058)
[2020-04-26 10:03:30 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80831)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80832)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80833)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80834)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80835)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80836)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80837)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80838)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80839)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80840)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80841)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80842)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80843)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80844)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80845)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80846)
[2020-04-26 12:03:49 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80847)
[2020-04-26 13:21:17 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80850)
[2020-04-26 13:21:17 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80855)
[2020-04-26 21:42:44 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80854)
[2020-04-26 21:42:45 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80878)
[2020-04-26 21:42:45 -0700] [10299] [CRITICAL] WORKER TIMEOUT (pid:80879)
2020-05-09 18:14:16,842 hit_not_registered_with_ad_server (1018) {'hitId': 'debug', 'assignmentId': 'debug', 'workerId': 'None', 'mode': 'sandbox'}
2020-05-09 18:18:03,282 hit_assign_worker_id_not_set_in_consent (1002) {'hitId': 'debug', 'assignmentId': 'debug', 'mode': 'sandbox'}
[2020-06-14 21:24:11 -0700] [50957] [CRITICAL] WORKER TIMEOUT (pid:50960)
[2020-06-16 12:50:28 -0700] [51524] [CRITICAL] WORKER TIMEOUT (pid:51527)
[2020-06-16 12:50:45 -0700] [53172] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2345, in _wrap_pool_connect
    return fn()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 364, in connect
    return _ConnectionFairy._checkout(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 778, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 495, in checkout
    rec = pool._do_get()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 140, in _do_get
    self._dec_overflow()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 137, in _do_get
    return self._create_connection()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 309, in _create_connection
    return _ConnectionRecord(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 440, in __init__
    self.__connect(first_connect_check=True)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 656, in __connect
    connection = pool._invoke_creator(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/strategies.py", line 114, in connect
    return dialect.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 490, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psycopg2/__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "ec2-54-88-130-244.compute-1.amazonaws.com" to address: nodename nor servname provided, or not known


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/experiment_server.py", line 48, in load
    return util.import_app("psiturk.experiment:app")
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/experiment.py", line 89, in <module>
    init_db()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/db.py", line 37, in init_db
    Base.metadata.create_all(bind=engine)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/sql/schema.py", line 4465, in create_all
    ddl.SchemaGenerator, self, checkfirst=checkfirst, tables=tables
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2103, in _run_visitor
    with self._optional_conn_ctx_manager(connection) as conn:
  File "/Users/dankmank/anaconda3/lib/python3.7/contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2095, in _optional_conn_ctx_manager
    with self._contextual_connect() as conn:
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2311, in _contextual_connect
    self._wrap_pool_connect(self.pool.connect, None),
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2349, in _wrap_pool_connect
    e, dialect, self
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1591, in _handle_dbapi_exception_noconnection
    sqlalchemy_exception, with_traceback=exc_info[2], from_=e
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2345, in _wrap_pool_connect
    return fn()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 364, in connect
    return _ConnectionFairy._checkout(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 778, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 495, in checkout
    rec = pool._do_get()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 140, in _do_get
    self._dec_overflow()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 137, in _do_get
    return self._create_connection()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 309, in _create_connection
    return _ConnectionRecord(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 440, in __init__
    self.__connect(first_connect_check=True)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 656, in __connect
    connection = pool._invoke_creator(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/strategies.py", line 114, in connect
    return dialect.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 490, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psycopg2/__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not translate host name "ec2-54-88-130-244.compute-1.amazonaws.com" to address: nodename nor servname provided, or not known

(Background on this error at: http://sqlalche.me/e/e3q8)
[2020-06-25 10:58:43 -0700] [7564] [CRITICAL] WORKER TIMEOUT (pid:7567)
[2020-06-27 09:48:30 -0700] [7564] [CRITICAL] WORKER TIMEOUT (pid:11721)
[2020-06-27 11:48:47 -0700] [7564] [CRITICAL] WORKER TIMEOUT (pid:14240)
[2020-06-27 11:48:48 -0700] [14250] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2345, in _wrap_pool_connect
    return fn()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 364, in connect
    return _ConnectionFairy._checkout(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 778, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 495, in checkout
    rec = pool._do_get()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 140, in _do_get
    self._dec_overflow()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 137, in _do_get
    return self._create_connection()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 309, in _create_connection
    return _ConnectionRecord(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 440, in __init__
    self.__connect(first_connect_check=True)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 656, in __connect
    connection = pool._invoke_creator(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/strategies.py", line 114, in connect
    return dialect.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 490, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psycopg2/__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "ec2-54-88-130-244.compute-1.amazonaws.com" to address: nodename nor servname provided, or not known


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/experiment_server.py", line 48, in load
    return util.import_app("psiturk.experiment:app")
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/experiment.py", line 89, in <module>
    init_db()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/db.py", line 37, in init_db
    Base.metadata.create_all(bind=engine)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/sql/schema.py", line 4465, in create_all
    ddl.SchemaGenerator, self, checkfirst=checkfirst, tables=tables
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2103, in _run_visitor
    with self._optional_conn_ctx_manager(connection) as conn:
  File "/Users/dankmank/anaconda3/lib/python3.7/contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2095, in _optional_conn_ctx_manager
    with self._contextual_connect() as conn:
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2311, in _contextual_connect
    self._wrap_pool_connect(self.pool.connect, None),
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2349, in _wrap_pool_connect
    e, dialect, self
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1591, in _handle_dbapi_exception_noconnection
    sqlalchemy_exception, with_traceback=exc_info[2], from_=e
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2345, in _wrap_pool_connect
    return fn()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 364, in connect
    return _ConnectionFairy._checkout(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 778, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 495, in checkout
    rec = pool._do_get()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 140, in _do_get
    self._dec_overflow()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 137, in _do_get
    return self._create_connection()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 309, in _create_connection
    return _ConnectionRecord(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 440, in __init__
    self.__connect(first_connect_check=True)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 656, in __connect
    connection = pool._invoke_creator(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/strategies.py", line 114, in connect
    return dialect.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 490, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psycopg2/__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not translate host name "ec2-54-88-130-244.compute-1.amazonaws.com" to address: nodename nor servname provided, or not known

(Background on this error at: http://sqlalche.me/e/e3q8)
[2020-07-02 10:37:40 -0700] [6188] [CRITICAL] WORKER TIMEOUT (pid:6191)
[2020-07-02 10:37:44 -0700] [6413] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2345, in _wrap_pool_connect
    return fn()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 364, in connect
    return _ConnectionFairy._checkout(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 778, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 495, in checkout
    rec = pool._do_get()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 140, in _do_get
    self._dec_overflow()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 137, in _do_get
    return self._create_connection()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 309, in _create_connection
    return _ConnectionRecord(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 440, in __init__
    self.__connect(first_connect_check=True)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 656, in __connect
    connection = pool._invoke_creator(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/strategies.py", line 114, in connect
    return dialect.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 490, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psycopg2/__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "ec2-54-88-130-244.compute-1.amazonaws.com" to address: nodename nor servname provided, or not known


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/experiment_server.py", line 48, in load
    return util.import_app("psiturk.experiment:app")
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/experiment.py", line 89, in <module>
    init_db()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/db.py", line 37, in init_db
    Base.metadata.create_all(bind=engine)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/sql/schema.py", line 4465, in create_all
    ddl.SchemaGenerator, self, checkfirst=checkfirst, tables=tables
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2103, in _run_visitor
    with self._optional_conn_ctx_manager(connection) as conn:
  File "/Users/dankmank/anaconda3/lib/python3.7/contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2095, in _optional_conn_ctx_manager
    with self._contextual_connect() as conn:
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2311, in _contextual_connect
    self._wrap_pool_connect(self.pool.connect, None),
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2349, in _wrap_pool_connect
    e, dialect, self
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1591, in _handle_dbapi_exception_noconnection
    sqlalchemy_exception, with_traceback=exc_info[2], from_=e
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2345, in _wrap_pool_connect
    return fn()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 364, in connect
    return _ConnectionFairy._checkout(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 778, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 495, in checkout
    rec = pool._do_get()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 140, in _do_get
    self._dec_overflow()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 137, in _do_get
    return self._create_connection()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 309, in _create_connection
    return _ConnectionRecord(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 440, in __init__
    self.__connect(first_connect_check=True)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 656, in __connect
    connection = pool._invoke_creator(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/strategies.py", line 114, in connect
    return dialect.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 490, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psycopg2/__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not translate host name "ec2-54-88-130-244.compute-1.amazonaws.com" to address: nodename nor servname provided, or not known

(Background on this error at: http://sqlalche.me/e/e3q8)
[2020-07-02 23:04:18 -0700] [6912] [CRITICAL] WORKER TIMEOUT (pid:6915)
[2020-07-02 23:07:15 -0700] [6912] [CRITICAL] WORKER TIMEOUT (pid:9297)
[2020-07-02 23:13:31 -0700] [6912] [CRITICAL] WORKER TIMEOUT (pid:9305)
[2020-07-02 23:25:15 -0700] [6912] [CRITICAL] WORKER TIMEOUT (pid:9324)
[2020-07-02 23:37:17 -0700] [6912] [CRITICAL] WORKER TIMEOUT (pid:9340)
[2020-07-02 23:53:00 -0700] [6912] [CRITICAL] WORKER TIMEOUT (pid:9354)
[2020-07-03 00:08:39 -0700] [6912] [CRITICAL] WORKER TIMEOUT (pid:9366)
[2020-07-03 00:16:28 -0700] [6912] [CRITICAL] WORKER TIMEOUT (pid:9377)
[2020-07-03 00:33:41 -0700] [6912] [CRITICAL] WORKER TIMEOUT (pid:9388)
[2020-07-03 01:34:41 -0700] [6912] [CRITICAL] WORKER TIMEOUT (pid:9399)
[2020-07-03 19:31:40 -0700] [10477] [CRITICAL] WORKER TIMEOUT (pid:10480)
[2020-07-03 19:32:14 -0700] [10477] [CRITICAL] WORKER TIMEOUT (pid:11304)
[2020-07-03 23:16:32 -0700] [10477] [CRITICAL] WORKER TIMEOUT (pid:11309)
[2020-07-04 01:28:28 -0700] [10477] [CRITICAL] WORKER TIMEOUT (pid:11672)
[2020-07-04 02:28:30 -0700] [10477] [CRITICAL] WORKER TIMEOUT (pid:11728)
[2020-07-04 05:32:28 -0700] [10477] [CRITICAL] WORKER TIMEOUT (pid:11738)
[2020-07-04 07:32:48 -0700] [10477] [CRITICAL] WORKER TIMEOUT (pid:11769)
[2020-07-04 09:33:07 -0700] [10477] [CRITICAL] WORKER TIMEOUT (pid:11806)
[2020-07-04 11:29:25 -0700] [10477] [CRITICAL] WORKER TIMEOUT (pid:11831)
[2020-07-04 12:25:41 -0700] [10477] [CRITICAL] WORKER TIMEOUT (pid:11851)
[2020-07-04 12:25:43 -0700] [11880] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2345, in _wrap_pool_connect
    return fn()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 364, in connect
    return _ConnectionFairy._checkout(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 778, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 495, in checkout
    rec = pool._do_get()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 140, in _do_get
    self._dec_overflow()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 137, in _do_get
    return self._create_connection()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 309, in _create_connection
    return _ConnectionRecord(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 440, in __init__
    self.__connect(first_connect_check=True)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 656, in __connect
    connection = pool._invoke_creator(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/strategies.py", line 114, in connect
    return dialect.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 490, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psycopg2/__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "ec2-54-88-130-244.compute-1.amazonaws.com" to address: nodename nor servname provided, or not known


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/experiment_server.py", line 48, in load
    return util.import_app("psiturk.experiment:app")
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/experiment.py", line 89, in <module>
    init_db()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/db.py", line 37, in init_db
    Base.metadata.create_all(bind=engine)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/sql/schema.py", line 4465, in create_all
    ddl.SchemaGenerator, self, checkfirst=checkfirst, tables=tables
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2103, in _run_visitor
    with self._optional_conn_ctx_manager(connection) as conn:
  File "/Users/dankmank/anaconda3/lib/python3.7/contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2095, in _optional_conn_ctx_manager
    with self._contextual_connect() as conn:
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2311, in _contextual_connect
    self._wrap_pool_connect(self.pool.connect, None),
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2349, in _wrap_pool_connect
    e, dialect, self
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1591, in _handle_dbapi_exception_noconnection
    sqlalchemy_exception, with_traceback=exc_info[2], from_=e
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2345, in _wrap_pool_connect
    return fn()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 364, in connect
    return _ConnectionFairy._checkout(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 778, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 495, in checkout
    rec = pool._do_get()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 140, in _do_get
    self._dec_overflow()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 137, in _do_get
    return self._create_connection()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 309, in _create_connection
    return _ConnectionRecord(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 440, in __init__
    self.__connect(first_connect_check=True)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 656, in __connect
    connection = pool._invoke_creator(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/strategies.py", line 114, in connect
    return dialect.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 490, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psycopg2/__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not translate host name "ec2-54-88-130-244.compute-1.amazonaws.com" to address: nodename nor servname provided, or not known

(Background on this error at: http://sqlalche.me/e/e3q8)
2020-07-06 00:18:34,101 Exception on /exp [GET]
Traceback (most recent call last):
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1284, in _execute_context
    cursor, statement, parameters, context
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 590, in do_execute
    cursor.execute(statement, parameters)
sqlite3.OperationalError: database is locked

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/user_utils.py", line 23, in new_func
    resp = make_response(func(*args, **kwargs))
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/experiment.py", line 454, in start_exp
    db_session.commit()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/scoping.py", line 163, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 1042, in commit
    self.transaction.commit()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 504, in commit
    self._prepare_impl()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 483, in _prepare_impl
    self.session.flush()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2523, in flush
    self._flush(objects)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2664, in _flush
    transaction.rollback(_capture_exception=True)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2624, in _flush
    flush_context.execute()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/unitofwork.py", line 422, in execute
    rec.execute(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/unitofwork.py", line 589, in execute
    uow,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 245, in save_obj
    insert,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 1083, in _emit_insert_statements
    c = cached_connections[connection].execute(statement, multiparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1020, in execute
    return meth(self, multiparams, params)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 298, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1139, in _execute_clauseelement
    distilled_params,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1324, in _execute_context
    e, statement, parameters, cursor, context
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1518, in _handle_dbapi_exception
    sqlalchemy_exception, with_traceback=exc_info[2], from_=e
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1284, in _execute_context
    cursor, statement, parameters, context
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 590, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) database is locked
[SQL: INSERT INTO turkdemo (uniqueid, assignmentid, workerid, hitid, ipaddress, browser, platform, language, cond, counterbalance, codeversion, beginhit, beginexp, endhit, bonus, status, mode, datastring) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('debugi7upY:debuglT6tK', 'debuglT6tK', 'debugi7upY', 'debug6iZbq', '127.0.0.1', 'chrome', 'macos', 'UNKNOWN', 0, 0, '1.0', '2020-07-06 00:18:29.054376', None, None, 0.0, 1, 'debug', None)]
(Background on this error at: http://sqlalche.me/e/e3q8)
2020-07-06 00:18:39,169 Exception on /exp [GET]
Traceback (most recent call last):
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1284, in _execute_context
    cursor, statement, parameters, context
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 590, in do_execute
    cursor.execute(statement, parameters)
sqlite3.OperationalError: database is locked

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/user_utils.py", line 23, in new_func
    resp = make_response(func(*args, **kwargs))
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/experiment.py", line 454, in start_exp
    db_session.commit()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/scoping.py", line 163, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 1042, in commit
    self.transaction.commit()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 504, in commit
    self._prepare_impl()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 483, in _prepare_impl
    self.session.flush()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2523, in flush
    self._flush(objects)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2664, in _flush
    transaction.rollback(_capture_exception=True)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2624, in _flush
    flush_context.execute()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/unitofwork.py", line 422, in execute
    rec.execute(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/unitofwork.py", line 589, in execute
    uow,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 245, in save_obj
    insert,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 1083, in _emit_insert_statements
    c = cached_connections[connection].execute(statement, multiparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1020, in execute
    return meth(self, multiparams, params)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 298, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1139, in _execute_clauseelement
    distilled_params,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1324, in _execute_context
    e, statement, parameters, cursor, context
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1518, in _handle_dbapi_exception
    sqlalchemy_exception, with_traceback=exc_info[2], from_=e
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1284, in _execute_context
    cursor, statement, parameters, context
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 590, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) database is locked
[SQL: INSERT INTO turkdemo (uniqueid, assignmentid, workerid, hitid, ipaddress, browser, platform, language, cond, counterbalance, codeversion, beginhit, beginexp, endhit, bonus, status, mode, datastring) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('debugi7upY:debuglT6tK', 'debuglT6tK', 'debugi7upY', 'debug6iZbq', '127.0.0.1', 'chrome', 'macos', 'UNKNOWN', 0, 0, '1.0', '2020-07-06 00:18:34.126903', None, None, 0.0, 1, 'debug', None)]
(Background on this error at: http://sqlalche.me/e/e3q8)
2020-07-06 00:18:51,024 Exception on /exp [GET]
Traceback (most recent call last):
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1284, in _execute_context
    cursor, statement, parameters, context
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 590, in do_execute
    cursor.execute(statement, parameters)
sqlite3.OperationalError: database is locked

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/user_utils.py", line 23, in new_func
    resp = make_response(func(*args, **kwargs))
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/psiturk/experiment.py", line 454, in start_exp
    db_session.commit()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/scoping.py", line 163, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 1042, in commit
    self.transaction.commit()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 504, in commit
    self._prepare_impl()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 483, in _prepare_impl
    self.session.flush()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2523, in flush
    self._flush(objects)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2664, in _flush
    transaction.rollback(_capture_exception=True)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2624, in _flush
    flush_context.execute()
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/unitofwork.py", line 422, in execute
    rec.execute(self)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/unitofwork.py", line 589, in execute
    uow,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 245, in save_obj
    insert,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 1083, in _emit_insert_statements
    c = cached_connections[connection].execute(statement, multiparams)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1020, in execute
    return meth(self, multiparams, params)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 298, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1139, in _execute_clauseelement
    distilled_params,
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1324, in _execute_context
    e, statement, parameters, cursor, context
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1518, in _handle_dbapi_exception
    sqlalchemy_exception, with_traceback=exc_info[2], from_=e
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1284, in _execute_context
    cursor, statement, parameters, context
  File "/Users/dankmank/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 590, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) database is locked
[SQL: INSERT INTO turkdemo (uniqueid, assignmentid, workerid, hitid, ipaddress, browser, platform, language, cond, counterbalance, codeversion, beginhit, beginexp, endhit, bonus, status, mode, datastring) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('debugi7upY:debuglT6tK', 'debuglT6tK', 'debugi7upY', 'debug6iZbq', '127.0.0.1', 'chrome', 'macos', 'UNKNOWN', 0, 0, '1.0', '2020-07-06 00:18:45.979011', None, None, 0.0, 1, 'debug', None)]
(Background on this error at: http://sqlalche.me/e/e3q8)
